### PR TITLE
Added: A tooltip for the locktime

### PR DESCRIPTION
**Pull request by killerswin2**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
    This adds to the locked tooltip, when the vehicle was locked. It is need for commanders, as they typically do not have access to the rpt logs to see lock times. Had to implement a way to get time-spans from sqf. I don't want to  do that again. This was done because we don't have ints in the engine.

### Please specify which Issue this PR Resolves.
N/A

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

**Comment by killerswin2** [comment]
Screenshot 2024-07-03 003453
Screenshot 2024-07-03 003700

File: A3A/addons/garage/Core/fn_reloadCategory.sqf [hunk 1]
```diff
@@ -29,7 +29,7 @@ lbClear _ctrl;
 private _selected = -1;
 private _HR_GRG_SelectedVehicles = [-1,-1,""];
 {
-    _y params ["_displayName", "_class", "_lockedUID", "_checkedOut", "", ["_lockName", ""]];
+    _y params ["_displayName", "_class", "_lockedUID", "_checkedOut", "", ["_lockName", ""], "", "_lockTime"];
```
Comment (jaj22) [inline A3A/addons/garage/Core/fn_reloadCategory.sqf:32]: _lockTime will be nil for old saves. Should handle that case somehow. Garage is mostly unscheduled so it doesn't fall over on a nil reference, but I couldn't figure what the results would be.

File: A3A/addons/garage/Core/fn_reloadCategory.sqf [hunk 2]
```diff
@@ -46,14 +46,21 @@ private _HR_GRG_SelectedVehicles = [-1,-1,""];
         };
         _ctrl lbSetPictureRight [_index, CheckOutIcon];
         _ctrl lbSetPictureRightColor [_index, _color];
-        _tooltipText = name (_checkedOut call BIS_fnc_getUnitByUID) +" "+( localize "STR_HR_GRG_Feedback_checkedOutToolTip" )+" ";
+        _tooltipText = format ["%1 %2 %3", name (_checkedOut call BIS_fnc_getUnitByUID), ( localize "STR_HR_GRG_Feedback_checkedOutToolTip" ), parseText "\n"];
```
Comment (jaj22) [inline A3A/addons/garage/Core/fn_reloadCategory.sqf:49]: The parseText doesn't actually do anything here. "\n" just passed through string -> text -> string, as it's not XML. You could just use `"%1 %2 \n"` as the format string with the same effect.

**Comment by jaj22** [comment]
Not sure why I let timespan_format through with capitalized units and no configurable minimum unit. Oh well.

**Review by jaj22 — APPROVED** [review]

**Review by HakonRydland — APPROVED** [review]
Forgot to add this review, lg for the garage bits, didnt bother reading the time formatter but it looks to work from the screenshot so good enough

**Pull request merged**